code/datums/effects/xeno_strains/execute_tag.dm

```diff
@@ -0,0 +1,41 @@
+/datum/effects/execute_tag
+	effect_name = "execute tag"
+	duration = null
+	flags = DEL_ON_DEATH | INF_DURATION
+
+
+/datum/effects/execute_tag/New(atom/affected, mob/from = null, last_dmg_source = null, zone = "chest", ttl = 35)
+	. = ..()
+
+	QDEL_IN(src, ttl)
+
+	if (ishuman(affected))
+		var/mob/living/carbon/human/affected_human = affected
+		affected_human.update_execute_hud()
```

Comment:
If `set_stat` is always updating the execute hud, do we need this here? Or do we need it there if we're doing it here?